### PR TITLE
initial sketch of the demo orchestrator

### PR DESCRIPTION
**Pull request by thomas-fossati**
Again; trying to repair the mess I did last night while pushing buttons carelessly...

### CODE REVIEW

File: demo/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+#   build:
+#     context: attester
+#     dockerfile: attester.Dockerfile
+#   stdin_open: true
+#   tty: true
+#   networks:
+#     demo:
+#   extra_hosts:
+#     - "pfe:${PFE_DEMO_ADDR}"
+#   container_name: relying-party
```
[the remark this answers is not in the document]
Comment (thomas-fossati) [reply demo/docker-compose.yml:56]: :thumbsup:

[the remark this answers is not in the document]
Comment (thomas-fossati) [reply demo/docker-compose.yml]: done in [87eaf98](https://github.com/CCC-Attestation/attested-tls-poc/pull/43/commits/87eaf9858e06f646b06ddd2243c2bc94d9a1791b)

File: demo/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      dockerfile: relying-party.Dockerfile
+    stdin_open: true
+    tty: true
+    networks:
+      demo:
+    extra_hosts:
+      - "vfe:${VFE_DEMO_ADDR}"
+    container_name: relying-party
+
+#  attester:
```
[the remark this answers is not in the document]
Comment (thomas-fossati) [reply demo/docker-compose.yml]: this is now uncommented -- I've rebased on Ionut's #40

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Pull request merged**